src/Config/ConfigLocator.php

```diff
@@ -280,7 +280,7 @@ public function addSitewideConfig($siteRoot)
         // Remember that we've seen this location.
         $this->siteRoots[] = $siteRoot;
 
-        $this->addConfigPaths(self::DRUPAL_CONTEXT, [ dirname($siteRoot) . '/drush', "$siteRoot/drush", "$siteRoot/sites/all/drush" ]);
+        $this->addConfigPaths(self::DRUPAL_CONTEXT, [ dirname($siteRoot) . '/drush', "$siteRoot/drush", "$siteRoot/sites/all/drush", "$siteRoot/sites/default" ]);
```

Comment:
Still -1 on this. We should not load config from sites/default when some other sites directory is bootstrapped. Use /drush/drush.yml at the project root or Drupal root.